src/ncp/ncp_base_ftd.cpp

```diff
@@ -57,6 +57,39 @@
 namespace ot {
 namespace Ncp {
 
+void NcpBase::HandleParentResponseInfo(otThreadParentResponseInfo *aInfo, void *aContext)
+{
+    VerifyOrExit(aInfo && aContext);
+
+    static_cast<NcpBase *>(aContext)->HandleParentResponseInfo(aInfo);
+
+exit:
+    return;
+}
+
+void NcpBase::HandleParentResponseInfo(otThreadParentResponseInfo *aInfo)
```

Comment:
Why not NcpBase::EncodeParentResponseInfo()?

param should be const?

Reply:
i will add the const to the pointer

Reply:
I'm fine either way. Will defer to @abtink / @jwhui.